src/ui/drawing/gdi/GDISurface.cpp

```diff
@@ -65,20 +65,20 @@ void GDISurface::SwitchFont(int nFont) const
     }
 }
 
+_Use_decl_annotations_
 void GDISurface::DrawImage(int nX, int nY, int nWidth, int nHeight, const ImageReference& pImage)
 {
-    auto hBitmap = ImageRepository::GetHBitmap(pImage);
-    if (!hBitmap)
+    if (pImage.GetData() == 0)
```

Comment:
This change prevents `GetHBitmap` from resolving the bitmap in the case where it hasn't already been loaded.

Reply:
I put it back the way it was. For `ImageRepository::GetHBitmap`, `ImageReference::SetData `had to be replaced a direct assignment to work.

Reply:
I think that was the root problem.